### ISSUE TITLE
'Lead' event runs in an error

### ISSUE THREAD

**Issue by Olgoetz** [RivercodeAB/facebook-conversion-api-nextjs#19]
```
fbEvent({
          eventName: "Lead",
          emails: [data.email],
          enableStandardPixel: false,
        });
```

as well as 

```
fbEvent({
          eventName: "Lead",
          emails: [data.email],
          enableStandardPixel: true,
        });
```
results in 
`POST http://localhost:3000/api/fb-events 400 (Bad Request) conversion-api.js:47 `

Is this a bug?

**Comment by ovdahlberg** [comment]
Hi @Olgoetz!

This should be fixed in 4.3.1. Please try and see if it works for you now :)

**Comment by Olgoetz** [comment]
everything good, thanks :)
